dashboard/config/locales/en.yml

```diff
@@ -692,82 +692,156 @@ en:
       body_short: "Design an app, code with blocks or JavaScript to make it work, then share your app in seconds."
       audience: 'Ages 13+'
       cta: 'Learn how'
-    dance:
-      title: 'Dance Party'
+      image_alt: 'A purple bunny is wearing a rocket powered backpack.'
```

Comment:
nit: Applies to many of these, do we care about active vs. passive voice for alt text? Ex. `A purple bunny wears a rocket-powered backpack."